### DISCUSSION TITLE
GitHub Topics Abuse for SEO purpose

### DISCUSSION THREAD

**Discussion by sagaratalatti** [community/community#70793] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

We’ve seen a huge amount of scams and hacks happening in the crypto space as the phishing script for draining web3 wallets is not available publicly. 

There’s one repository that always keeps popping up on GitHub as soon as it’s removed from GitHub after being removed. The vendor is creating a ReadMe file only to sell a crypto drainer script for $2,000  he often uses topics to get the most buyers. I have been reporting for past 15 days but the vendor pops back everytime with a new repository.

[Redacted due to platform safety]

I did some research around the topics he is using and found out they’re all ranked one in Google search engine. 

[Redacted due to platform safety]
While reporting this repository I have always mentioned about the topics and how it’s being abused by some hackers and sell phishing codes publicly. GitHub being one of the leading collaborative platform for programming has still not figured out to fix these issues. 

They remove such topics from GitHub and ban their use for repositories.

**Comment by sagaratalatti** [comment]
As soon as today’s report got all his repositories removed this person came back with even more duplicate repositories 

...
...
...
...
...
...
...
...
...

This is how easy it’s to create multiple accounts post abusive repositories in misuse of Google rankings of GitHub topics.

**Comment by Kartikay-101** [comment]
Response

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @sagaratalatti, 

Thank you so much for reporting this and continually doing so - I know that's frustrating. I've passed this along to our experts to review further and take a closer look. I removed the screenshots and am going to close this discussion as we don't want anyone accidentally clicking on those links. Appreciate you helping keep GitHub a safe space!

**Reply by sagaratalatti** [reply]
Another repository from the similar seller has popped up again seems like he's using a different approach since he has changed some keywords for his repository description.

![image](https://github.com/community/community/assets/13254211/4bc66833-72ae-4444-b38d-b066ed8c42fb)

He's using the following topics which have ranked GitHub #1 on google search engine.

drainer
crypto-stealer
nft-drainer
eth-drainer
nft-stealer
crypto-drainer
best-wallet-drainer
drainer-script
wallet-drainer
metamask-wallet-drainer
erc20-drainer
metamask-drainer
trustwallet-drainer
bnb-drainer
usdt-drainer
metamask-stealer
trust-wallet-drainer
multichain-drainer

I think these sellers won't stop unless these topics are removed/banned on GitHub

**Reply by sagaratalatti** [reply]
Some topics were left out and the drainer is up again on GitHub:

[Removed Link]

[Removed Link]

Topics that still work: 

ethereum-drainer 
crypto-drainer-scripts
metamask-hack
